index.bs

```diff
@@ -837,6 +837,12 @@ When an {{XRReferenceSpace}} is requested, the user agent MUST <dfn>create a ref
 
   1. Let |session| be the {{XRSession}} object that requested creation of a reference space.
   1. Let |type| be set to the {{XRReferenceSpaceType}} passed to {{requestReferenceSpace()}}.
+  1. If |type| is {{bounded-floor}} or {{unbounded}}, and |session|'s [=XRSession/mode=] is {{XRSessionMode/inline}}, return <code>null</code>
+  1. If |type| is {{bounded-floor}} or {{unbounded}}, and |session|'s [=XRSession/mode=] is not {{XRSessionMode/inline}} the user agent may choose to return <code>null</code> if such experiences are not supported by the underlying [=/XR device=]
+  1. If |type| is {{local-floor}} or {{local}}, and |session|'s [=XRSession/mode=] is {{XRSessionMode/inline}} the user agent may choose to return <code>null</code> if such experiences are not supported by the underlying [=/XR device=] in non-immersive sessions
+
+      NOTE: All other combinations of {{XRReferenceSpaceType}}s and [=XRSession/mode=]s are guaranteed by this spec, provided the [=XRSession/mode=] itself is supported.
```

Comment:
I can't imagine placing a note in the middle of an algorithm like this formats nicely? It should definitely be moved outside the algorithm.

Reply:
Notes like this seem to work fine, they get indented. A couple specs do this

Reply:
Well color me surprised! Today I learned something new.